src/demo/share/jfc/Font2DTest/README.txt

```diff
@@ -129,23 +111,3 @@ that are within the selected range. Third option, "Print all text..."
 is similar, and it will print all lines of text that user has put in.
 
 ====================================================================
-
-Known Problems:
-
-- When a PostScript font is used, the characters may extend beyond the
-enclosing grid or zoom rectangle. This is due to the problem with
-FontMetrics.getMaxAscent() and getMaxDescent() functions; the functions
-do not always return the right values for PostScript fonts.
-
-- There are still some bugs around the error handling.
-Most of these problems will usually get fixed when some parameters
-are changed, or the screen is refreshed.
```

Comment:
Are the 1st 2 known problems are not there now?

Reply:
I do not know of problems with getting metrics for Postscript fonts and I have no idea what the 2nd comment means and it may be ancient history which is why I want to clean it up.